pkgs/applications/virtualization/rust-hypervisor-firmware/x86_64-unknown-none.json

```diff
@@ -0,0 +1,20 @@
+{
+  "llvm-target": "x86_64-unknown-none",
```

Comment:
is there a reason for this very custom target?
shouldn't that be just the UEFI target?

Reply:
aaaaaaah I forgot that it's firmware, nevermind me